src/dde-file-manager-lib/models/computermodel.cpp

```diff
@@ -61,14 +62,17 @@ ComputerModel::ComputerModel(QObject *parent)
     m_nitems = 0;
     m_appEntryWatcher.reset(fileService->createFileWatcher(nullptr, DUrl(APPENTRY_ROOT), this));
 
+    bool disabledMyDirectories = DFMApplication::genericAttribute(DFMApplication::GA_HiddenMyDirectories).toBool();
```

Comment:
How should I make them re-display without restarting dde-file-manager after I changed the config?
Your solution is not COMPLETE so I will NOT approve to merge.
Maybe you should watch the change of the attribute and reload the model when the value changed, that will be more acceptable.